src/hotspot/share/oops/instanceKlass.hpp

```diff
@@ -1046,8 +1046,8 @@ class InstanceKlass: public Klass {
            vtable_length +
            itable_length +
            nonstatic_oop_map_size +
-           (is_interface ? (int)sizeof(Klass*)/wordSize : 0) +
-           (is_inline_type ? (int)sizeof(InlineKlassFixedBlock) : 0));
+           (is_interface ? (int)sizeof(Klass*) / wordSize : 0) +
+           (is_inline_type ? (int)sizeof(InlineKlassFixedBlock) / wordSize : 0));
```

Comment:
Do we have an `x / wordSize` helper function anywhere? Too much inline arithmetic makes it hard to maintain, imo.